app/components/autocomplete/index.js

```diff
@@ -39,18 +53,17 @@ export default class Autocomplete extends Component {
     };
 
     render() {
+        const container = this.props.isSearch ? style.searchContainer : style.container;
         return (
             <View>
-                <View style={style.container}>
+                <View style={container}>
```

Comment:
Should we just pass in the style as a prop so that the parent of this (in this case the search textbox) can override it without needing something very specific to the search? I'm thinking about once we go to support editing posts or editing the channel header